Clinical-Genomics/cg
# Update Scout Config Generation with Chromograph Support

This PR updates generation av Scout configuration regarding Chromograph png-files.

### How to prepare for test
- [x] ssh to hasta (depending on type of change)
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh feature_chromograph-CG_201121`

### How to test
1. Log onto hasta
2. Run a Scout configuration command: `cg upload scout -p <id>`
if Chromograph files exist an entry should be created with the correct files (i.e. file path missing suffix). If not, no files should be added.

```
samples.chromograph_images.{
    upd_regions,
    upd_sites,
    coverage,
    autozygosity
}
```

<strike>IDs to test with: _none at the moment_ ☹️</strike>
Test ID: `deepbeagle` 😀



<img width="859" alt="image" src="https://user-images.githubusercontent.com/1150065/99880105-a0df7f80-2c11-11eb-8580-91699adc4b8e.png">


## Review
- [x] tests executed by ML
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
